contracts/governance/StkTruToken.sol

```diff
@@ -74,6 +76,7 @@ contract StkTruToken is VoteToken, ClaimableContract, ReentrancyGuard {
     event Cooldown(address indexed who, uint256 endTime);
     event CooldownTimeChanged(uint256 newUnstakePeriodDuration);
     event UnstakePeriodDurationChanged(uint256 newUnstakePeriodDuration);
+    event FeePayerWhitelistingStatusChanged(address payer, bool status);
```

Comment:
Should payer be an `indexed` argument? Not sure it really matters, but it looks like you've been doing that almost everywhere else with addresses.

Reply:
this doesn't get called a lot & I think you can only index a certain number of params per contract